router/src/server.rs

```diff
@@ -111,8 +111,9 @@ request_body = CompatGenerateRequest,
 responses(
 (status = 200, description = "Generated Text",
 content(
-("application/json" = Vec<GenerateResponse>),
-("text/event-stream" = StreamResponse),
+(Vec<GenerateResponse> = "application/json"),
+(Vec<GenerateResponse> = "application/json"),
```

Comment:
Duplicated here!

```suggestion
```

Reply:
thanks, avoided these issues/changes, by avoiding bumping the utopia version